scipy/stats/tests/generate_fisher_exact_results_from_r.R

```diff
@@ -0,0 +1,82 @@
+# This R script generates the Python file fisher_exact_results_from_r.py.
```

Comment:
This is very nice.
Maybe add a small comment on how to run e.g. `$ Rscript fisher_exact_results_from_r.py`

Reply:
Done.